lib/smarter_csv/smarter_csv.rb

```diff
@@ -186,6 +186,35 @@ def SmarterCSV.process(input, options={}, &block)   # first parameter: filename
     end
   end
 
+  # SmarterCSV.write(rows, out, header = nil)
+  #   `rows`   -> Array of hashes representing each row.
+  #   `out`    -> The filename to write the data to.
+  #   `header` -> Array of column names as strings or symbols. Optional. If it
```

Comment:
might be better to define this with a hash, ` {'a' =>'name1', 'b' => 'name2', ...}`  ?